packages/react-server/src/logger/logger.ts

```diff
@@ -3,7 +3,10 @@ import {Context, Request} from 'koa';
 import chalk from 'chalk';
 import {KoaNextFunction} from '../types';
 
-export const LOGGER = Symbol('logger');
+// Have to cast as `any` or typescript will complain
+// about using a "unique symbol as an index type".
+// https://github.com/microsoft/TypeScript/issues/1863
+export const LOGGER: string = Symbol('logger') as any;
```

Comment:
Open to other ways around this. It is a pattern we use a lot so unfortunate that we have to cast.